backend/src/main/java/com/repository/meeting/MeetingRepository.java

```diff
@@ -1,9 +1,29 @@
 package com.repository.meeting;
 
+import com.dto.query.MeetingWithInvite;
 import com.model.meeting.Meeting;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface MeetingRepository extends JpaRepository<Meeting, Integer> {
+
+    @Query(value = "SELECT meeting.* FROM meeting JOIN invite USING(meeting_id)" +
+            " WHERE room_id = :room_id AND meeting_id != :meeting_id",
+            nativeQuery = true)
+    List<Meeting> findAllByRoomId(@Param("room_id")Integer roomId, @Param("meeting_id") Integer meetingId);
+
+    @Query(value = "SELECT meeting.* FROM meeting JOIN invite USING(meeting_id)" +
+            " WHERE employee_id = :employee_id AND meeting_id != :meeting_id AND invite_accept = 1",
+            nativeQuery = true)
+    List<Meeting> findAllEmployeeMeetings(@Param("employee_id") Integer employeeId, @Param("meeting_id") Integer meetingId);
```

Comment:
**Not critical**  
Incorrect naming - your query returns meetings that are accepted by the employee.
So, another possible naming - "findAllEmployeeAcceptedMeetings".

Reply:
Okay. Will fix it into next pull request.